src/unibasicTemplates/findByIds.njk

```diff
@@ -39,6 +33,12 @@ if recordIdsCount eq 0 then
   go returnFromSub
 end
 
+* get the projections from the options
```

Comment:
This was an existing bug that was exposed by changing projection to be an array or null.  There is a type check above this block that is intended to validate that the list of ids is an array.  However, when projection got added, it used the same `type` variable that the `ids` property get did, which means the type validation was running against the projection array rather than the ids array.  I simply relocated this `udoGetProperty` call to ensure that the proper `type` variable is being inspected.